wrappers/python/tests/layer/test_prod.py

```diff
@@ -0,0 +1,156 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/tests/layer/test_prod.py
+# Test for nntile.layer.prod
+#
+# @version 1.0.0
+
+import time
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+import nntile
+from nntile.layer.act import Act
+from nntile.layer.linear import Linear
+from nntile.layer.prod import Prod
+from nntile.loss import Frob
+from nntile.model.base_model import BaseModel
+from nntile.tensor import Tensor_fp32, TensorMoments, TensorTraits, notrans
+
+
+class ToyFC_Torch(nn.Module):
+    def __init__(self, input_dim, hidden_dim):
+        super().__init__()
+        self.lin1 = nn.Linear(input_dim, hidden_dim, bias=False)
+        self.act = nn.ReLU()
+        self.lin2 = nn.Linear(input_dim, hidden_dim, bias=False)
+        self.lin3 = nn.Linear(hidden_dim, input_dim, bias=False)
+
+    def forward(self, x):
```

Comment:
Is this test for gated MLP layer? It does not seem to be a simple `prod` function tester